### PR TITLE
Added more grids

### PR DESCRIPTION
**Pull request by valentinavolgina2**
- Added more grids using [resource 1](https://www.whatdowedoallday.com/tic-tac-toe-variations/) and r[esource 2](https://en.wikipedia.org/wiki/Tic-tac-toe_variants)
- Adjusted cell width for icon grids and the cards cross axis count for better fit

https://user-images.githubusercontent.com/64236396/233450529-f3c7f433-0375-437b-be6f-c02b9e238bf7.mov

### CODE REVIEW

File: lib/grids_page.dart [hunk 1]
```diff
@@ -16,6 +16,12 @@ class GridsPage extends StatelessWidget {
 
   final List boardsList = boardTemplates.keys.toList();
 
+  int getCrossAxisCountByWidth(width) {
```
Comment (polina-c) [inline lib/grids_page.dart:19]: Should this method be private in this library?
Avoid methods that starts with `get`: https://dart.dev/guides/language/effective-dart/design#avoid-starting-a-method-name-with-get

**Review by polina-c — APPROVED** [review]
One comment then lgtm.

File: lib/model/boards.dart [hunk 2]
```diff
@@ -14,7 +14,32 @@ final boardTemplates = <String, Board>{
...
+  '5 X 5': [
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
+  ],
+  'Four in a row': [
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
+    [CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty, CellState.empty],
```
Comment (polina-c) [inline lib/model/boards.dart:32]: Why such name?

File: lib/grids_page.dart (hunk 1 above)
Comment (valentinavolgina2) [reply lib/grids_page.dart]: Fixed

File: lib/model/boards.dart (hunk 2 above)
Comment (valentinavolgina2) [reply lib/model/boards.dart:32]: The first player to get four-in-a-row vertically, horizontally, or diagonally wins (instead of classic three)

**Pull request merged**